java-manta-client/src/main/java/com/joyent/manta/client/multipart/AbstractMultipartManager.java

```diff
@@ -197,7 +198,8 @@ public PART uploadPart(final UPLOAD upload,
      */
     abstract PART uploadPart(UPLOAD upload,
                              int partNumber,
-                             HttpEntity entity) throws IOException;
+                             HttpEntity entity,
+                             HttpContext context) throws IOException;
```

Comment:
Forgot the JavaDoc, thanks for calling it out. Regarding the nullability of `HttpContext` it's fine to pass null since the only param that is required is `HttpUriRequest`. `ServersideMultipartManagerIT#canUploadWithSinglePartByteArray` validates that this is the case. See [CloseableHttpClient's source for reference](canUploadWithSinglePartByteArray)